addons/website/static/src/scss/website.scss

```diff
@@ -36,6 +36,7 @@ $-seen-urls: ();
     @include print-variable('has-customized-13-0-color-system', $o-has-customized-13-0-color-system);
 
     // 1) Handle default values
+    @include print-variable('paragraph-margin-bottom', $paragraph-margin-bottom);
```

Comment:
This is not a default value, it is the exact value we set it up to. Move this to `5)` like the rest please :pray: